alacritty/src/display.rs

```diff
@@ -369,6 +378,61 @@ impl Display {
         let glyph_cache = &mut self.glyph_cache;
         let size_info = self.size_info;
 
+        // Check grid damage
+        let damage: Option<Vec<Rect>> = if self.damage_supported {
+            let (width, height, cell_width, cell_height, padding_x, padding_y) =
+                size_info.into_u32();
+
+            if self.fully_damaged {
+                // We need to fully damaged, so let's clear damage and stop
+                // here.
+                terminal.get_damage();
```

Comment:
We need to clear damage and not to get it, so adding clear_damage on term will make more sense here(it also faster).